### PR TITLE
Lmgtfy

### PR DESCRIPTION
**Pull request by wimo7083**

### CODE REVIEW

File: pybot/endpoints/slack/messages.py [hunk 1]
```diff
@@ -96,3 +60,55 @@ def not_bot_delete(event: Message):
...
+        channel=event["channel"],
+        text=f"OC-Community-Bot is a community led project\n <{BOT_URL}|source> ",
+    )
+
+    await app.plugins["slack"].api.query(methods.CHAT_POST_MESSAGE, data=response)
+
+
+async def search_link(event: Message, app: SirBot):
+    # TODO: add ability to parse @user_id and ping the user
+    unparsed_message =  event.get('text')
```
Comment (AllenAnthes) [inline pybot/endpoints/slack/messages.py]: Nit: Unnecessary extra space here

File: pybot/endpoints/slack/messages.py [hunk 2]
```diff
@@ -96,3 +60,55 @@ def not_bot_delete(event: Message):
...
+    response = dict(
+        channel=event["channel"],
+        text="DuckDuckGo Search Results",
+        attachments=[dict(title=f'{unparsed_message}', 
+                          text=f'https://lmgtfy.com/?s=d&q={'+'.join(unparsed_message.split(' '))}')]
```
Comment (AllenAnthes) [inline pybot/endpoints/slack/messages.py]: Currently includes the `!lmgtfy` text in the query.  Is that intentional or should it be pulled out?

File: pybot/endpoints/slack/messages.py [hunk 3]
```diff
@@ -96,3 +60,55 @@ def not_bot_delete(event: Message):
         logger.debug(
             f"Exception thrown while logging message_deleted. Event: {event} || Error: {e}"
         )
+
+
+
```
Comment (AllenAnthes) [inline pybot/endpoints/slack/messages.py]: Nit: Remove one of the newlines here

**Comment by aaron-suarez** [comment]
@wimo7083 If you want to reopen this feel free

**Pull request closed without merging**